server/service/handler.go

```diff
@@ -99,15 +99,6 @@ func MakeHandler(svc fleet.Service, config config.FleetConfig, logger kitlog.Log
 	r.Use(publicIP)
 
 	attachFleetAPIRoutes(r, svc, config, logger, limitStore, fleetAPIOptions)
-
-	// Results endpoint is handled different due to websockets use
-
-	// TODO: this would not work once v1 is deprecated - note that the handler too uses the /v1/ path
-	// and this routes on path prefix, not exact path (unlike the authendpointer struct).
-	r.PathPrefix("/api/v1/fleet/results/").
-		Handler(makeStreamDistributedQueryCampaignResultsHandler(svc, logger)).
-		Name("distributed_query_results")
```

Comment:
The endpoint's name will now be auto-generated like any other, so the prometheus metric will now be under that new name - the same was true for all other endpoints when we moved to the auto-generation of names, so I'm guessing it's not that big of a deal, just that the websocket endpoint being special, it could be a more scrutinized metric?

Reply:
Hm, not sure, the websocket endpoint is not super documented anyway. We should be fine with making this fit the same pattern as the rest.